packages/camera/camera_avfoundation/example/ios/RunnerTests/AvailableCamerasTest.m

```diff
@@ -55,29 +54,26 @@ - (void)testAvailableCamerasShouldReturnAllCamerasOnMultiCameraIPhone {
   }
   OCMStub([discoverySessionMock devices]).andReturn([NSArray arrayWithArray:cameras]);
 
-  // Set up method call
-  FlutterMethodCall *call = [FlutterMethodCall methodCallWithMethodName:@"availableCameras"
-                                                              arguments:nil];
-
-  __block id resultValue;
-  [camera handleMethodCallAsync:call
-                         result:^(id _Nullable result) {
-                           resultValue = result;
-                           [expectation fulfill];
-                         }];
+  __block NSArray<FCPPlatformCameraDescription *> *resultValue;
+  [camera
+      availableCamerasWithCompletion:^(NSArray<FCPPlatformCameraDescription *> *_Nullable result,
+                                       FlutterError *_Nullable error) {
+        XCTAssertNil(error);
+        resultValue = result;
+        [expectation fulfill];
+      }];
+  [self waitForExpectationsWithTimeout:30 handler:nil];
 
   // Verify the result
-  NSDictionary *dictionaryResult = (NSDictionary *)resultValue;
   if (@available(iOS 13.0, *)) {
-    XCTAssertTrue([dictionaryResult count] == 4);
+    XCTAssertEqual(resultValue.count, 4);
```

Comment:
This is just a quality of life improvement for when the test fails. (I had a failure initially because I didn't add the `wait`, so `resultValue` was nil, and I got a failure with no actual info.)